Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -46,6 +50,32 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager) {
         masterTalons = new HashMap<XCANTalon, BaseDriveSubsystem.MotionRegistration>();
         masterTalons.put(leftMaster, new MotionRegistration(0, 1, -1));
         masterTalons.put(rightMaster, new MotionRegistration(0, 1, 1));
+        
+        
+        leftMaster.createTelemetryProperties("LeftDriveMaster");
+        rightMaster.createTelemetryProperties("RightDriveMaster");
+        
+        leftTicksPerFiveFeet = propManager.createPersistentProperty("leftDriveTicksPer5Feet", 0);
+        rightTicksPerFiveFeet = propManager.createPersistentProperty("rightDriveTicksPer5Feet", 0);
+        
+    }
+    
+    public double ticksToInches(double ticks) {
+        double tpi = ticksPerInch();
+        
+        if (tpi != 0) {
+            return (1.0 / ticksPerInch()) * ticks;
+        }
+        return 0;
+    }
+    
+    public double inchesToTicks(double inches) {
+        // this is taking the average of left and right, then dividing by 60 inches (5 feet)
+        return ticksPerInch() * inches;
+    }
+    
+    private double ticksPerInch() {
+        return ((leftTicksPerFiveFeet.get() + rightTicksPerFiveFeet.get()) / 120.0);
```

Comment:
"120" requires me to think to figure out if that's the right conversion. Can we break it up into a named variable and two separate arithmetic ops?

Reply:
Sure, good idea not to inline so much stuff.